src/language/worker.ts

```diff
@@ -27,11 +26,7 @@ const getContent = ({ type, printable }: EvaluationResult): Document => {
     : getEJSON(printable);
 };
 
-export const getLanguage = (
-  evaluationResult: EvaluationResult
-): 'json' | 'plaintext' => {
-  const content = getContent(evaluationResult);
-
+export const getLanguage = (content: any): 'json' | 'plaintext' => {
```

Comment:
could also potentially be `unknown`?

Reply:
It should default to `plaintext` if this is not a `json`. This defines the type of the virtual document we open with a playground result.

Reply:
was referring to `any` => `unknown`, I think we don't know much about the content then right?

Reply:
Ah, I see! I thought you were talking about the return type.